default.config.yml

```diff
@@ -152,23 +152,15 @@ php_custom_config:
     option: "sendmail_path"
     value: "Mailhog sendmail"
   - section: "xdebug"
-    option: "xdebug.remote_port"
-    value: "9001"
+    option: "xdebug.client_port"
+    value: "9003"
```

Comment:
Is the port change really needed?

Reply:
Not really, it's to follow what xdebug is doing.

I think that's not really a problem because PHPStorm allows 2 ports:

<img width="543" alt="CleanShot 2021-01-16 at 14 06 28@2x" src="https://user-images.githubusercontent.com/523981/104812689-4bd07c00-5804-11eb-9bbb-af7499b5e56a.png">

Reply:
ok.